src/certs/ca/cert/v1.rs

```diff
@@ -9,10 +9,16 @@ const NAPLES_ARK_SIG: &[u8] = include_bytes!("../../../../tests/naples/ark.cert.
 const NAPLES_ARK: Preamble = Preamble {
     ver: 1u32.to_le(),
     data: Data {
-        kid: 122178821951678173525318614033703090459u128.to_le(),
-        sid: 122178821951678173525318614033703090459u128.to_le(),
+        kid: [
+            0x1b, 0xb9, 0x87, 0xc3, 0x59, 0x49, 0x46, 0x06, 0xb1, 0x74, 0x94, 0x56, 0x01, 0xc9,
+            0xea, 0x5b,
+        ],
+        sid: [
+            0x1b, 0xb9, 0x87, 0xc3, 0x59, 0x49, 0x46, 0x06, 0xb1, 0x74, 0x94, 0x56, 0x01, 0xc9,
+            0xea, 0x5b,
+        ],
```

Comment:
I think a byte string literal here will be more concise: https://doc.rust-lang.org/reference/tokens.html#byte-string-literals